docker/wordpress/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM wordpress:5.8.3
+
+RUN apt update && apt install -y libxml2-dev
+RUN docker-php-ext-install soap
```

Comment:
Did we verify if we needed these?

Reply:
Nope. I can verify that today.

Reply:
Yeah they don't appear to be necessary, so I've removed them and amended the commit which added the Dockerfile.